src/components/Functions/ServoCalibration.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+    import {writable} from "svelte/store";
+    import {openSidecar} from "../../utils/sidecar";
+    import ToastStore from "../../stores/toast";
+    import Select from "../Input/Select.svelte";
+    import SuspenseButton from "../Input/Button/SuspenseButton.svelte";
+
+    const state = writable({
+            form: {
+                calibrationTimer: 5,
+                rightHand: true,
+            },
+            loading: false,
+    });
+
+    const beginCalibration = async (extend) => {
+        try {
+            $state.loading = true;
+            const result = await openSidecar('sidecar', 'functions_servotest', {
+                extend,
+                right_hand: $state.form.rightHand
+            });
+            ToastStore.addToast(ToastStore.severity.SUCCESS, result.pop());
+            $state.loading = false;
+        } catch (e) {
+            $state.loading = false;
+            console.trace(e);
+            ToastStore.addToast(ToastStore.severity.ERROR, 'Error starting servo calibration. Make sure that the driver is running: ' + e);
+        }
+
+    };
+</script>
+
+<tr>
+    <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+        Servo Testing for Manual Calibration
+    </td>
+    <td class="px-6 py-4 text-sm text-gray-500">
+        <div class="mb-3">
+            <Select options={[{title: 'Left Hand', value: false}, {title: 'Right Hand', value: true}]}
+                    onSelectItemChanged={v => $state.form.rightHand = v}
+                    defaultValue={true} label="For Hand"/>
+        </div>
+        Calibrate your servos for Force Feedback. The buttons on the right give the options to either extend the servos
+        fully, or retract them fully. As a guide, when the servos extended all the way, place the servo horn to where you
+        want the fingers to be fully restricted. Then, retract the servos. They should not impede on any finger movement.
```

Comment:
```suggestion
        <b>Calibrate your servos for Force Feedback</b>
        <br />
        The buttons on the right give the options to either extend the servos fully, or retract them fully. As a guide, when
        the servos extended all the way, place the servo horn to where you want the fingers to be fully restricted. Then,
        retract the servos. They should not impede on any finger movement.
```
Similar readability change as in auto `PoseCalibration.svelte`